crates/worker/src/partition/state_machine/command_interpreter.rs

```diff
@@ -441,6 +452,14 @@ where
     ) -> Result<(FullInvocationId, SpanRelation), Error> {
         let related_span = metadata.journal_metadata.span_context.as_parent();
 
+        self.kill_child_invocations(
+            &full_invocation_id,
+            state,
+            effects,
+            metadata.journal_metadata.length,
+        )
+        .await?;
+
```

Comment:
Shouldn't we enqueue the `abort_invocation` command before the kill_child_invocations?

Actually even with the `fail_invocation` probably should come before the `abort_invocation`?

Reply:
Since everything is executed by the same thread and the individual effects don't interfere with each other, I believe that the order of the effects does not really matter.